src/frontend/screens/Game/GamePage/index.tsx

```diff
@@ -427,15 +438,15 @@ export default function GamePage(): JSX.Element | null {
               <Anticheat gameInfo={gameInfo} />
               <div className="buttonsWrapper">
                 {is_installed && (
-                  <>
-                    <button
-                      disabled={isReparing || isMoving || isUpdating}
-                      onClick={handlePlay()}
-                      className={`button ${getPlayBtnClass()}`}
-                    >
-                      {getPlayLabel()}
-                    </button>
-                  </>
+                  <button
+                    disabled={
+                      isReparing || isMoving || isUpdating || isUninstalling
+                    }
+                    onClick={handlePlay()}
+                    className={`button ${getPlayBtnClass()}`}
+                  >
+                    {getPlayLabel()}
+                  </button>
```

Comment:
I'm just adding the `|| isUninstalling` and removing the wrapping `<></>` fragment, but the diff makes it look like more things changed